fastseq/optimizer/fairseq/el_attention_optimizer.py

```diff
@@ -0,0 +1,1459 @@
+
+"""Apply the beam search optimizations to fairseq-v0.9.0"""
```

Comment:
Add copyright and modify this comment. (EL-attention related code?)

\#Copyright (c) Microsoft Corporation.
\#Licensed under the MIT License.

Reply:
Done